specification/Metadata/Semantics/README.md

```diff
@@ -0,0 +1,73 @@
+# Cesium Metadata Semantic Reference
+
+This document defines a general set of semantics for 3D Tiles and glTF. Tileset authors may define their own application- or domain-specific semantics separately.
+
+Semantics describe how properties should be interpreted. For example, an application that sees the `HORIZON_OCCLUSION_POINT` semantic would use the property for horizon occlusion culling as defined below.
```

Comment:
A good example, but how many readers will know about horizon culling.  Many readers will be from Mod Sim.  Why not use an example that speaks to them and to developers in general?